msg/json/en.json

```diff
@@ -1,7 +1,7 @@
 {
 	"@metadata": {
 		"author": "Ellen Spertus <ellen.spertus@gmail.com>",
-		"lastupdated": "2024-04-16 23:19:53.668551",
+		"lastupdated": "2023-12-08 18:42:04.679586",
```

Comment:
this is also a really weird change lol but assuming it's just from the way we just straight up replace this file instead of actually merging it

Reply:
Reverted as per your second comment.